### DISCUSSION TITLE
Are "bots" and their maintainers Test contributors?

### DISCUSSION THREAD

**Discussion by javiercasares** [WordPress/hosting-handbook#241] [Open topics]
Should we ask the Test team to give a Test badge for the Bots and their maintainers? (e.g https://profiles.wordpress.org/wpsabot/ this profile is empty, but “contributes” here https://make.wordpress.org/hosting/test-results/).

Moreover, ask for that to the developers involved in the test-runner and test-reporter.

**Comment by pfefferle** [comment]
I like the idea!

**Comment by rogermedia** [comment]
It seems like it would help get more Bots running.

**Comment by noruzzamans** [comment]
Sounds great!

**Comment by desrosj** [comment]
A few weeks ago we discussed this in the Hosting team meeting in Slack. During that discussion we clarified the problem that this was trying to solve: It's not immediately obvious that these bot accounts are not regular user accounts, which can be confusing.

Historically, companies and businesses are not recognized within the Credits API. Contributions belong to individuals and not the companies that sponsor them or enable them to contribute. I think the same should extend to these test bot accounts. Badges are a way to recognize human contributions, so I don't think we should consider a badge for bot accounts.

However, I am in favor of any of the following to try and avoid confusion:
- Denoting the account as a host test bot account in some way on their profile. 
- Listing the bot's reports on their profile in the activity feed.
- Ability to link a bot account to another WP.org profile or 5FTF pledge organization (not sure how that assignment works behind the scenes).

These things also provide additional recognition/celebrate participation in the hosting tests. Here are a few examples of what I have in mind.
<img width="1049" alt="Screenshot 2024-10-12 at 9 41 42 AM" src="https://github.com/user-attachments/assets/bb3bd80a-9c67-4931-b3fd-60fc99f78e2a">
<img width="1405" alt="Screenshot 2024-10-12 at 9 54 47 AM" src="https://github.com/user-attachments/assets/2a0973ad-b3ae-44a6-8ff7-b003ea3e329b">
